amount_in_words_invoice/migrations/16.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    reset_templates(cr)
+
+
+def reset_templates(cr):
+    """Reset templates with source files to update their translations
+
+    Currently, the migration of the database does not update the values or translations of the templates
+    generated by code, so they had to be deleted or reloaded to be regenerated. But that did not update the
+    translations so they also had to be reloaded by forcing the overwriting of existing terms.
+
+    Odoo has already a method to perform both actions which is to reset the template.
+    This way it is only necessary to select all the templates generated by code and use the method to solve
+    both problems.
+    """
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    templates = env["mail.template"].search([("template_fs", "!=", False)])
```

Comment:
This is reloading all templates. However, since this is not a migration script for a customer app, we shouldn't touch other unrelated records.

Also adjust docstring accordingly.